src/Bicep.Core.Samples/Files/InvalidDisableNextLineDiagnosticsDirective_CRLF/main.diagnostics.bicep

```diff
@@ -0,0 +1,9 @@
+#disable-next-line
+//@[18:18) [BCP226 (Error)] Expected at least one diagnostic code at this location. Valid format is "#disable-next-line diagnosticCode1 diagnosticCode2 ..." (CodeDescription: none) ||
+param storageAccount1 string = 'testStorageAccount'
+//@[6:21) [no-unused-params (Warning)] Parameter "storageAccount1" is declared but never used. (CodeDescription: bicep core(https://aka.ms/bicep/linter/no-unused-params)) |storageAccount1|
+#  disable-next-line  no-unused-params
+//@[0:1) [BCP001 (Error)] The following token is not recognized: "#". (CodeDescription: none) |#|
+//@[0:1) [BCP007 (Error)] This declaration type is not recognized. Specify a parameter, variable, resource, or output declaration. (CodeDescription: none) |#|
+param storageAccount2 string = 'testStorageAccount'

```

Comment:
A couple of other test cases that would be good to add:
```
/* comment before */#disable-next-line foo
```
```
#disable-next-line/* comment between */foo
```
```
//#disable-next-line foo
```
```
#disable-next-line
foo
```

```
#madeup-directive
```
```
var terminatedWithDirective = 'foo' #disable-next-line no-unused-params
```

Reply:
Updated here:
https://github.com/Azure/bicep/commit/08c8ad4a560e8c6fcafa9a9e5eae60b875d84333